frontend/src/components/ShootHibernation/GHibernationScheduleEvent.vue

```diff
@@ -30,31 +30,29 @@ SPDX-License-Identifier: Apache-2.0
           />
         </v-col>
         <v-col cols="2">
-          <v-text-field
+          <g-time-text-field
             ref="wakeUpTime"
             v-model="wakeUpTime"
             color="primary"
             label="Wake up at"
             :error-messages="getErrorMessages('wakeUpTime')"
-            type="time"
             clearable
             variant="underlined"
             @blur="touchIfNothingFocused"
-            @input="onInputWakeUpTime"
+            @update:model-value="onInputWakeUpTime"
```

Comment:
use `model-value` inststead of `v-model` when using `@update:model-value` for both occurrences

Reply:
? which occurrence?

Reply:
<g-time-text-field /> component is used twice in this file (line 33 and line 46). Both use v-model

Reply:
Ah okay now I think I understand what you mean. Changing `v-model` to `model-value` would require further changes as we require the two-way data binding in this case. Of course we could handle this in the `@update:model-value` event handler but then we should change this everywhere. Right now we use this event handler in lots of places together with v-model, mostly as replacement for `@input` events that now behave differently.